README.md

```diff
@@ -86,7 +86,8 @@ route:
   group_wait: 30s
 
   # When the first notification was sent, wait 'group_interval' to send a batch
-  # of new alerts that started firing for that group.
+  # of new alerts that started firing for that group. Alert resolution notifications
```

Comment:
This doesn't read cleanly. I'd suggest something like "to send the next batch of alerts for that group" that implicitly covers both.

Reply:
> to send the next batch of alerts for that group

not so clean also.
Maybe

`to send the next batch of notifications for that group` ? 
because *alert* means something like *firing*, not *resolved* :thinking:

Reply:
That doesn't work. There's no such thing as a a batch of notifications, only a group of alerts.

Reply:
ugh. It is totally confusing. Any ideas about improving the docs? I really want to clarify, that `group_interval' affects both firing and resolved notifications

Reply:
As you wrote here https://www.robustperception.io/whats-the-difference-between-group_interval-group_wait-and-repeat_interval

> Instead we wait for the group_interval since the last notification was sent to the group, and then send all alerts firing (and any resolved alerts) to the receiver.

It is good enough. But it is not official docs :D

Reply:
A related issue: the repeat_interval is also confusing.

In some case, I received notification more than once in a repeat interval. This is surprising.